src/home/room_preview.rs

```diff
@@ -278,9 +281,14 @@ impl Widget for RoomPreviewContent {
                 }
             }
 
-            let unread_badge = self.view(id!(unread_badge));
+            let unread_badge = self.view(id!(unread_badge)); 
 
             if room_info.num_unread_messages > 0 {
+                if room_info.num_unread_metions > 0{
+                    unread_badge.apply_over(cx, live!{draw_bg: {highlight: 1.0}});
+                }else{
+                    unread_badge.apply_over(cx, live!{draw_bg: {highlight: 0.0}});
+                }
                 if room_info.num_unread_messages > 99 {
```

Comment:
nit: syntax spacing:
```suggestion
                if room_info.num_unread_metions > 0 {
                    unread_badge.apply_over(cx, live!{ draw_bg: { highlight: 1.0 }});
                } else {
                    unread_badge.apply_over(cx, live!{ draw_bg: { highlight: 0.0 }});
                }
                if room_info.num_unread_messages > 99 {
```

Reply:
Thank you for your review!